### ISSUE TITLE
Didn't work well in test

### ISSUE THREAD

**Issue by 139419qw** [pleuren/vggface-finetune#1]
Hi, I have done your finetune method in my dataset. It seems goes well on training: acurracy=1, very low loss.But on my test set,outcomes seems to be constant.Could u plz help me with that?

**Comment by bleuren** [comment]
Let me check your steps.
How did you do it?
